sklearn/preprocessing/_data.py

```diff
@@ -60,22 +60,27 @@
 ]
 
 
-def _handle_zeros_in_scale(scale, copy=True):
+def _handle_zeros_in_scale(scale, copy=True, constant_mask=None):
```

Comment:
maybe add a short doc on what is a parameter `constant_mask`

Reply:
Done in 139d37d